codemods/object-keys/index.js

```diff
@@ -1,5 +1,6 @@
 import jscodeshift from "jscodeshift";
 import { removeImport } from "../shared.js";
+import { log } from "console";
```

Comment:
Was this intentional? 🤔 I don't see it used anywhere, shall we remove it? 🙂

Reply:
I just went ahead and removed it :)

Reply:
Oh it definitely was not, thanks 😅